lib/net/peer.js

```diff
@@ -1900,6 +1905,40 @@ class Peer extends EventEmitter {
     return false;
   }
 
+  /**
+   * Increase dosscore on peer.
+   * See: https://github.com/btcsuite/btcd/blob/master/connmgr/dynamicbanscore.go
+   * @param {Number} dosScore
+   * @param {Number} time
+   * @returns {Boolean}
+   */
+
+  increaseDos(dosScore, time) {
+    // dosScore is a time-bound transient banscore useful to detect DOS
+    // attacks by repeated requests for a message (e.g. getproof).  Calling
+    // increaseBan with a dosScore multiple times will increase dosScore in a
```

Comment:
nit: @tuxcanfly this comment mentions "increaseBan" instead of "increaseDos."

Reply:
Thanks, fixed.